deps/v8/test/cctest/interpreter/test-bytecode-generator.cc

```diff
@@ -2884,129 +2884,129 @@ TEST(PrivateAccessorAccess) {
-  BytecodeExpectationsPrinter printer(CcTest::isolate());
-
-  const char* snippets[] = {
-      "{\n"
-      "  class A {\n"
-      "    static #a() { return 1; }\n"
-      "  }\n"
-      "}\n",
-
-      "{\n"
-      "  class A {\n"
-      "    static get #a() { return 1; }\n"
-      "  }\n"
-      "}\n",
-
-      "{\n"
-      "  class A {\n"
-      "    static set #a(val) { }\n"
-      "  }\n"
-      "}\n",
-
-      "{\n"
-      "  class A {\n"
-      "    static get #a() { return 1; }\n"
-      "    static set #a(val) { }\n"
-      "  }\n"
-      "}\n",
-
-      "{\n"
-      "  class A {\n"
-      "    static #a() { }\n"
-      "    #b() { }\n"
-      "  }\n"
-      "}\n"};
-
-  CHECK(CompareTexts(BuildActual(printer, snippets),
-                     LoadGolden("StaticPrivateMethodDeclaration.golden")));
-  i::FLAG_harmony_private_methods = old_methods_flag;
-}
-
-TEST(StaticPrivateMethodAccess) {
-  bool old_methods_flag = i::FLAG_harmony_private_methods;
-  i::FLAG_harmony_private_methods = true;
-  InitializedIgnitionHandleScope scope;
-  BytecodeExpectationsPrinter printer(CcTest::isolate());
-  printer.set_wrap(false);
-  printer.set_test_function_name("test");
-
-  const char* snippets[] = {
-      "class A {\n"
-      "  static #a() { return 1; }\n"
-      "  static test() { return this.#a(); }\n"
-      "}\n"
-      "\n"
-      "var test = A.test;\n"
-      "test();\n",
-
-      "class B {\n"
-      "  static #b() { return 1; }\n"
-      "  static test() { this.#b = 1; }\n"
-      "}\n"
-      "\n"
-      "var test = B.test;\n"
-      "test();\n",
-
-      "class C {\n"
-      "  static #c() { return 1; }\n"
-      "  static test() { this.#c++; }\n"
-      "}\n"
-      "\n"
-      "var test = C.test;\n"
-      "test();\n",
-
-      "class D {\n"
-      "  static get #d() { return 1; }\n"
-      "  static set #d(val) { }\n"
-      "\n"
-      "  static test() {\n"
-      "    this.#d++;\n"
-      "    this.#d = 1;\n"
-      "    return this.#d;\n"
-      "  }\n"
-      "}\n"
-      "\n"
-      "var test = D.test;\n"
-      "test();\n",
-
-      "class E {\n"
-      "  static get #e() { return 1; }\n"
-      "  static test() { this.#e++; }\n"
-      "}\n"
-      "var test = E.test;\n"
-      "test();\n",
-
-      "class F {\n"
-      "  static set #f(val) { }\n"
-      "  static test() { this.#f++; }\n"
-      "}\n"
-      "var test = F.test;\n"
-      "test();\n",
-
-      "class G {\n"
-      "  static get #d() { return 1; }\n"
-      "  static test() { this.#d = 1; }\n"
-      "}\n"
-      "var test = G.test;\n"
-      "test();\n",
-
-      "class H {\n"
-      "  set #h(val) { }\n"
-      "  static test() { this.#h; }\n"
-      "}\n"
-      "var test = H.test;\n"
-      "test();\n"};
-
-  CHECK(CompareTexts(BuildActual(printer, snippets),
-                     LoadGolden("StaticPrivateMethodAccess.golden")));
-  i::FLAG_harmony_private_methods = old_methods_flag;
-}
+// TEST(StaticPrivateMethodDeclaration) {
```

Comment:
Rather than comment out the entire test you could prefix the test name with `DISABLED_` and the test runner will ignore it by default: https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#temporarily-disabling-tests

That would allow the test to still be run by passing the `--gtest_also_run_disabled_tests` CLI flag or `GTEST_ALSO_RUN_DISABLED_TESTS` environment variable.

Reply:
Thanks for the hint, I changed it accordingly.